Strip mage targets from Magefile (#412)
elastic/integrations: CONTRIBUTING.md

recommendations and tricks. Please consider this section as a live document that may evolve in the future, depending
on the business or technical requirements for the entire platform (Elastic Package Registry, Elastic Agent and Kibana).

### elastic-package

> The [elastic-package](https://github.com/elastic/elastic-package) is a command line tool, written in Go, used for developing Elastic packages. It can help you lint,

Reviewer: ```suggestion
[elastic-package](https://github.com/elastic/elastic-package) is a command line tool, written in Go, used for developing Elastic packages. It can help you lint,
```

   Explanation: it's much faster to rebuild and restart the container with the Package Registry, than work with
   mounted volumes.

#### Testing

> The `elastic-package` provides different types of test runners. Review [howto](https://github.com/elastic/elastic-package/tree/master/docs/howto) guides

Reviewer: ```suggestion
`elastic-package` provides different types of test runners. Review [howto](https://github.com/elastic/elastic-package/tree/master/docs/howto) guides
```

> to find the best option for testing packages.

Reviewer: "option" makes it sound like there is choice between various test methods, implying the user only needs to choose one. 

```suggestion
to learn about the various methods for testing packages.
```
